lib/danger/commands/runner.rb

```diff
@@ -56,7 +57,8 @@ def self.options
         ["--head=[master|dev|stable]", "A branch/tag/commit to use as the head"],
         ["--fail-on-errors=<true|false>", "Should always fail the build process, defaults to false"],
         ["--dangerfile=<path/to/dangerfile>", "The location of your Dangerfile"],
-        ["--danger_id=<id>", "The identifier of this Danger instance"]
+        ["--danger_id=<id>", "The identifier of this Danger instance"],
+        ["--new-comment", "Makes the build process ignore other Danger instances, resulting in a new comment. Overrides '--danger_id'."]
```

Comment:
IMHO a little bit too implementation specific 😅? How about something like `Makes Danger post a new comment for each build`😄?